pkgs/by-name/bu/bun/hooks/default.nix

```
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenvNoCC,
+  jq,
+  moreutils,
+  cacert,
+  makeSetupHook,
+  bun,
+}:
+
+{
+  fetchDeps =
+    {
+      hash ? "",
+      pname,
+      workspaces ? [ ],
+      preInstall ? "",
+      installFlags ? [ ],
+      ...
+    }@args:
+    let
+      args' = builtins.removeAttrs args [
+        "hash"
+        "pname"
+      ];
+    in
+    stdenvNoCC.mkDerivation (
+      finalAttrs:
+      (
+        args'
```

```suggestion
    stdenvNoCC.mkDerivation (
      args'
```
`finalAttrs` isn't actually used anywhere. We don't need it